FirstProject/css/main.css

```diff
@@ -0,0 +1,149 @@
+
+/*Header section*/
+header {
+    background-image: url(../img/feedback-banner.jpg);
+    width: 100%;
+}
+
+.left_image {
+    height: 290px;
+}
+
+.right_description {
+    font-weight: bold;
+}
+
+
+.container-fluid {
+    background-color: lightgrey;
+}
+
+/*Body section*/
+body {
+    overflow-x:hidden;
+}
+
+a {
+    text-decoration: none!important;
+    cursor: pointer;
+    color: black!important;
+    font-weight: 700;
+    font-size: 24px;
+}
+
+a:hover {
+    border-bottom: 4px solid blue;
+    border-bottom-width: 7px;
+}
+
+
+th, td {
+    padding-left: 55px!important;
+    font-size: 20px;
+}
+
+td {
+    border-bottom: 1px solid black;
+}
+
+/*Footer section*/
+footer.footer_section {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    text-align: center;
+}
+
+ul {
+    list-style-type: none;
+}
+
+li.bullets {
+    padding: 10px;
+}
+
+.submit_button {
+    float: right;
+    margin-top: 30px;
+    padding: 15px 40px;
+    margin-right: 100px;
+    border: 2px solid #03A9F4;
+    color: white;
+    font-size: 16px;
+    background-color: #03A9F4;
+    font-weight: bold;
+    border-radius: 4px;
+}
+
+form.language_form {
```

Comment:
Same question about extra specificity. Applicable throughout this file, in fact.